rust/src/cliwrap/kernel_install.rs

```diff
@@ -64,13 +67,20 @@ fn redo_systemctl_wrap() -> Result<()> {
     Ok(())
 }
 
-#[context("Running dracut")]
-fn run_dracut(kernel_dir: &str) -> Result<()> {
+#[context("Generate initfs")]
+fn generate_initfs(kernel_dir: &str) -> Result<()> {
     let root_fs = Utf8Dir::open_ambient_dir("/", cap_std::ambient_authority())?;
     let tmp_dir = tempfile::tempdir()?;
     let tmp_initramfs_path = tmp_dir.path().join("initramfs.img");
 
-    let cliwrap_dracut = Utf8Path::new(cliwrap::CLIWRAP_DESTDIR).join("dracut");
+    let mut initfs_tool = String::new();
+    let initoverlayfs_path = Path::new(INITOVERLAY_INSTALL_CMD);
+    if initoverlayfs_path.exists() {
+        initfs_tool = INITOVERLAY_INSTALL_CMD.to_string()
+    } else {
+        initfs_tool = "dracut".to_string()
+    }
+    let cliwrap_dracut = Utf8Path::new(cliwrap::CLIWRAP_DESTDIR).join(initfs_tool);
```

Comment:
I don't think this is right, because we aren't cliwrapping the initoverlay command.  So combining with the above, we should just directly assign to `dracut_path` below.